batches/BatchListComponent.tsx

```diff
@@ -8,11 +8,21 @@ import { RootState } from '../store/store';
 import { changeBatch } from '../store/actions';
 import { style } from '../global_styles';
 
+/**
+ * @typedef {Object} VisibleBatch
+ * @property {number} index
+ * @property {string} info
+ */
 interface VisibleBatch {
 	index: number;
 	info: string;
 }
 
+/**
+ * Renders the Batch List for the user
+ * @param {*} param0
```

Comment:
so it doesn't need to accept navigation and route? it can accept anything?

Reply:
That's is how it was written. I can add an interface like the one in Batch Home to give the illusion of definition but they both are essentially the same in that it will take anything